test/e2e/self_provisioned_elasticsearch_test.go

```diff
@@ -3,19 +3,19 @@
 package e2e
 
 import (
+	"context"
 	goctx "context"
 	"os"
-	"strconv"
 	"strings"
 	"testing"
 
+	"github.com/sirupsen/logrus"
```

Comment:
Something is odd with our linter/formatter. From time to time, it splits imports without reason... 

Could you place it back in the third-party block?

Reply:
@jpkrohling done